lib/ansible/plugins/filter/urlsplit.py

```diff
@@ -5,6 +5,55 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+DOCUMENTATION=r'''
+  name: urlsplit
+  version_added: "2.4"
+  short_description: get components from URL
+  description:
+    - Given an URL, spiti it into it's component parts
+  positional: _input, query
+  options:
+    _input:
+      description: url string to split
+      type: str
+      required: true
+    query:
+      description: specify a single compoenent to return
+      type: str
+      choices: ["fragment", "hostname", "netloc", "password",  "path",  "port",  "query", "scheme",  "username"]
+'''
+
+EXAMPLES=r'''
+
+    {{ "http://user:password@www.acme.com:9000/dir/index.html?query=term#fragment" | urlsplit }}
```

Comment:
```suggestion
    parts: '{{ "http://user:password@www.acme.com:9000/dir/index.html?query=term#fragment" | urlsplit }}'
```
Otherwise this fragment does not parse as YAML. (The yamllint sanity test expects it to do so.)

I guess same for the others.